web-test-runner.config.js

```diff
@@ -88,7 +88,7 @@ export default {
         threshold: {
             statements: 98.5,
             branches: 95.78,
-            functions: 97.8,
+            functions: 96.8,
```

Comment:
I'd be interested to know where the coverage actually dropped, but doesn't seem like that big of a deal for the overall win.